airflow/providers/google/cloud/triggers/cloud_composer.py

```diff
@@ -78,3 +78,71 @@ async def run(self):
                 "operation_done": operation.done,
             }
         )
+
+
+class CloudComposerAirflowCLICommandTrigger(BaseTrigger):
```

Comment:
This class is missing tests

Reply:
I have added a test, thank you for reviewing!